examples_src/modify-features.js

```diff
@@ -19,7 +19,8 @@ var raster = new ol.layer.Tile({
 var vector = new ol.layer.Vector({
   source: new ol.source.Vector({
     url: 'data/geojson/countries.geojson',
-    format: new ol.format.GeoJSON()
+    format: new ol.format.GeoJSON(),
+    wrapX: false
```

Comment:
As above, this needs to be in `examples` instead of `examples_src` (and a 3-way merge should do it).